.travis.yml

```diff
@@ -3,5 +3,18 @@ language: python
 python:
   - "3.6"
 
+env:
+  - GITHUB_USER=ovv GITHUB_TOKEN=43fd59047bfab99ed4f357471ba624b27ec5f846
+
 script:
   - python setup.py flake8
+  - pip install dothub
+  - yes | dothub repo --owner pyslackers --repository sirbot-pythondev push
+
+deploy:
+  provider: script
+  script: travis/dothub.sh
+  on:
+#    repo: pyslackers/sirbot-pythondev
```

Comment:
I think since these comments aren't indented this will break the yaml (pycharm does these wrong :( )